package.json

```diff
@@ -2,10 +2,15 @@
   "name": "example-app",
   "version": "0.0.0",
   "private": true,
+  "config" : {
+    "port" : 3000
+  },
   "scripts": {
-    "start": "DEBUG=example-app PORT=3000 NODE_ENV=production node bin/www",
+    "start": "DEBUG=example-app NODE_ENV=production node bin/www",
+    "clean": "rm -rf www",
```

Comment:
where is this being used? could probably use it here https://github.com/jomaxx/example-app/pull/6/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R11